config/strides/synapse-login-strides-params.yaml

```diff
@@ -29,5 +29,6 @@ sceptre_user_data:
       Value: >-
         '[
           {"teamId":"3407239","roleArn":"arn:aws:iam::423819316185:role/ServiceCatalogEndusers"},
+          {"teamId":"3438230","roleArn":"arn:aws:iam::423819316185:role/ServiceCatalogEndusers"},
```

Comment:
could you please add a comment (just above) to indicate which team this maps to?

Reply:
Done!

Reply:
actually,  i don't think end users (human accounts) should share the same role as  service accounts.  What if we need to give service accounts different access permissions than humans?  I think it would be better to create a different role for service accounts,  something like this..
```
{"teamId":"3438230","roleArn":"arn:aws:iam::423819316185:role/ServiceCatalogServiceUsers"}
```

Reply:
@zaro0508 , Team 3438230 is not a team for 'service accounts' per se but for Sage internal users (as suggested by the name).  That why it maps to the IAM role for Sage users.

Reply:
Seems like we are already sharing roles among multiple teams which isn't good however I agree that we can deal with that as a separate issue.